docs: add hscs workshop - index page + setup only

bguiz opened the pull request:
**Description**:

This PR a a new HSCS workshop to the tutorials section:

* index page
* setup page

Not yet included in this PR:

* solidity page
* hedera sdk js page
* hardhat page
* outro page

**Related issue(s)**:

* based on: https://github.com/hedera-dev/hedera-smart-contracts-workshop/
* discussion: https://swirldslabs.slack.com/archives/C040CSLT7M0/p1690187855884669

**Notes for reviewer**:

* more pages (as indicated in the description) will be added in subsequent PRs
* this was run as a 2 hour live workshop as part of an (external) hackathon, and the video recording from that may be clipped into smaller segments and placed into each of these pages

**Checklist**

- [x] Documented (Code comments, README, etc.)
- [x] Tested (unit, integration, etc.)

theekrystallee commented on tutorials/smart-contracts/hscs-workshop/README.md:
```diff
@@ -0,0 +1,96 @@
+description: >-
+  Hedera Smart Contract Service workshop.
+  Smart contracts are a means to enable custom logic and processing in a DLT.
+  Developers can harness their power to build their own decentralised applications (DApps).
```
Update all instances of British English to American English standards to keep it consistent with the rest of the documentation by replacing all `'S'` to `'Z'`

Example: decentralised should be decentralized with a `'Z'`

theekrystallee commented on tutorials/smart-contracts/hscs-workshop/README.md:
```diff
@@ -0,0 +1,96 @@
+- Syntax of Solidity, a programming language used to write smart contracts
+- Using the Solidity compiler
+- Using Hedera SDK JS to deploy smart contracts onto Hedera networks
+- Using Hedera SDK JS to interact with smart contracts on Hedera networks
```
Consider removing line 18 and updating line 17 to: 

Using Hedera SDK JS to interact and deploy smart contracts to the Hedera network

theekrystallee commented on tutorials/smart-contracts/hscs-workshop/README.md:
```diff
@@ -0,0 +1,96 @@
+- Using Hedera SDK JS to deploy smart contracts onto Hedera networks
+- Using Hedera SDK JS to interact with smart contracts on Hedera networks
+- Using Hardhat + EthersJs to deploy smart contracts onto Hedera networks
+- Using Hardhat + EthersJs to interact with smart contracts on Hedera networks
```
Consider removing line 20 and updating line 19 to: 

Using Hardhat + EthersJs to interact and deploy smart contracts to the Hedera network

theekrystallee commented on tutorials/smart-contracts/hscs-workshop/README.md:
```diff
@@ -0,0 +1,96 @@
+- Google chrome installed: [Chrome](https://www.google.com/chrome/)
+- Internet connection
+
+Optional system setup:
```
This can be captured using the "Expandable" GitBook feature as a drop-down.

bguiz replied:
Since there's only one optional item, I've just combined it with the main list of pre-requisites

theekrystallee commented on tutorials/smart-contracts/hscs-workshop/setup.md:
```diff
@@ -0,0 +1,259 @@
+
+To do so, we will utilise something called a **seed phrase**,
+which is a sequence of selected dictionary words chosen at random.
+This process is defined in BIP-39.
```
I suggest hyperlinking all instances of BIPs to their respective page/repo

bguiz replied:
Yep, linked inline + also in the info box below.

theekrystallee commented on tutorials/smart-contracts/hscs-workshop/setup.md:
```diff
@@ -0,0 +1,259 @@
+
+Visit the [Hedera Portal](https://portal.hedera.com/) to get started.
+
+* (1) Create a Testnet account.
```
Steps need to be bulleted for unordered steps and numbered for ordered steps, not both.

bguiz replied:
Good catch! Changing all to numbered only.

bguiz commented:
Closing this PR, it is superseded by [PR#421](https://github.com/hashgraph/hedera-docs/pull/421)
which includes all 6 pages.
@theekrystallee addressed your review comments in a commit on that one (`0f7614157a702601f178d28d7ad56875e8684970`).
